### DISCUSSION TITLE
Problems with some..

### DISCUSSION THREAD

**Discussion by mwedmark** [frntc/SIDKick-pico#21] [Q&A] [answered]
It looks like some of my built SIDkick-pico is not booting/working like they should in C64 although there are built with care. I tried to Ohm for shorts between close pins, found none. Programming works 100% and my feeling is that the Pi Pico is working as it should. Could it be the overclocking to 300Mhz failing on some chips? Are there any good way of testing for this? Is it possible to clock it at, say 200Mhz to test this theory but still have sound genereated? Thanks!

**Comment by frntc** [comment, marked as answer]
On Forum64 there are >>200 SIDKick picos and I haven't got feedback about Picos which were not able to go at 300 MHz (and other projects do the same), so I'd say unlikely, yet still out of specs. I assume you soldered the Picos? Otherwise the obvious way to test is swapping Picos and PCBs... another test you could do is: compile some blink example from the SDK and add overclocking.

The SKpico firmware does not run at 200 MHz, as the bus timings would need adjustment, and also the rest would most likely not work out of the box.

**Reply by mwedmark** [reply]
OK thanks! Yea a blink example with several different overclock's is a good way to test this theory, thanks!
